galaxy/templates/rbac-job.yaml

```diff
@@ -10,8 +10,11 @@ rules:
 - apiGroups: [""]
   resources: ["pods", "pods/log"]
   verbs: ["get", "list", "watch"]
-- apiGroups: ["batch", "extensions"]
-  resources: ["jobs"]
+- apiGroups: ["batch", "extensions", "networking.k8s.io"]
+  resources: ["jobs", "ingresses"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
```

Comment:
Do we need to update/patch ingresses? I was thinking, create and delete only?

Reply:
Do we need update/patch for jobs if you know? (I don't think we do?) If not, i'm pro removing them altogether

Reply:
Actually it might be used to scale down a job now that I think about it. I'll just remove it for services and ingresses for now

Reply:
Done in https://github.com/galaxyproject/galaxy-helm/pull/282/commits/50ecf41874b5929a4a9e5b3b894ade752db443d8